Formula/node@4.rb

```diff
@@ -50,6 +50,15 @@ def install
     system "make", "install"
   end
 
+  def post_install
+    return if build.without? "npm"
+
+    (lib/"node_modules/npm/npmrc").atomic_write <<~EOS
+      prefix = #{HOMEBREW_PREFIX}
+      python = /usr/bin/python\n
```

Comment:
I think the \n actually is not needed. Just noticed that after pulling node :(

Reply:
I left is that way for consistency, because we added the end with new line to the npmrc for ages in the main node formula see: https://github.com/Homebrew/homebrew-core/commit/102c3214c94fb6dff539c9f3ea5aab572e208ebb#diff-d3ef2414308e30df24a517ca9a1324eeL105 (and in the versioned formulae before the npm resource removal too).

Reply:
right. but the heredoc already does that, no? you're now ending in \n\n